app/src/main/res/values/styles.xml

```diff
@@ -175,7 +175,7 @@
         <item name="android:textAllCaps">false</item>
         <item name="android:textSize">14sp</item>
         <item name="android:textColor">@color/toggle_text_color</item>
-        <item name="android:drawableTint" tools:targetApi="m">@color/toggle_text_color</item>
+        <item name="drawableTint" tools:targetApi="m">@color/toggle_text_color</item>
```

Comment:
Does `app:` work here? The change appears to just silence the check for older devices.

Reply:
Apparently removing the `android:` is the way to do this... https://stackoverflow.com/questions/39336054/using-app-namespace-in-style

BUT we should probably also remove the `tools:targetApi="m"` to make this more clear